packages/mongodb-verified-answers/.env.example

```diff
@@ -0,0 +1,18 @@
+PORT=3000
+OPENAI_ENDPOINT=https://<resource_name>.openai.azure.com/
+OPENAI_API_KEY=<api_key>
+OPENAI_EMBEDDING_DEPLOYMENT=<deployment_name>
+OPENAI_EMBEDDING_MODEL=text-embedding-ada-002
+OPENAI_EMBEDDING_MODEL_VERSION=2023-03-15-preview
+OPENAI_CHAT_COMPLETION_DEPLOYMENT=<deployment_name>
+OPENAI_CHAT_COMPLETION_MODEL_VERSION=<model_version>
+MONGODB_CONNECTION_URI=<connection_uri>
+MONGODB_DATABASE_NAME=<docs-chatbot-ENV>
+VECTOR_SEARCH_INDEX_NAME=<index_name>
+NODE_ENV=development
+# Connection string for DevCenter
+DEVCENTER_CONNECTION_URI="mongodb+srv://<username>:<password>@devhub-cluster.sewho.mongodb.net/?retryWrites=true&w=majority"
+# For connecting to MongoDB University Data API
+UNIVERSITY_DATA_API_KEY=abc123
+# For use with GitHubDataSource. The token only needs read access to the repo.
+GITHUB_ACCESS_TOKEN=ghp_TOKENTOKENTOKEN
```

Comment:
```suggestion
OPENAI_ENDPOINT=https://<resource_name>.openai.azure.com/
OPENAI_API_KEY=<api_key>
OPENAI_EMBEDDING_DEPLOYMENT=<deployment_name>
OPENAI_EMBEDDING_MODEL=text-embedding-ada-002
OPENAI_EMBEDDING_MODEL_VERSION=2023-03-15-preview
MONGODB_CONNECTION_URI=<connection_uri>
MONGODB_DATABASE_NAME=<docs-chatbot-ENV>
```

I think we don't need a good chunk of these. Might want to double check that this list is correct - I based it off of the single call to `assertEnvVars`.